pandas/core/series.py

```diff
@@ -5004,6 +5004,42 @@ def reindex(  # type: ignore[override]
             tolerance=tolerance,
         )
 
+    @overload
+    def rename_axis(  # type: ignore[override]
+        self,
+        mapper: IndexLabel | lib.NoDefault = ...,
+        *,
+        index=...,
+        axis: Axis = ...,
+        copy: bool = ...,
+        inplace: Literal[True] = ...,
```

Comment:
The default value of `inplace` is `False`, so you cannot have `= ...` here.

Reply:
This is clearly not my best day. Fixed.